libs/community/langchain_community/vectorstores/yellowbrick.py

```diff
@@ -178,27 +402,104 @@ def from_texts(
         metadatas: Optional[List[dict]] = None,
         connection_string: str = "",
         table: str = "langchain",
+        schema: Optional[str] = None,
+        drop: Optional[bool] = False,
         **kwargs: Any,
     ) -> Yellowbrick:
         """Add texts to the vectorstore index.
         Args:
             texts: Iterable of strings to add to the vectorstore.
             metadatas: Optional list of metadatas associated with the texts.
-            connection_string: URI to Yellowbrick instance
+            connection_opt: URI to Yellowbrick instance or a pg connection
             embedding: Embedding function
             table: table to store embeddings
             kwargs: vectorstore specific parameters
         """
-        if connection_string is None:
-            raise ValueError("connection_string must be provided")
         vss = cls(
             embedding=embedding,
             connection_string=connection_string,
             table=table,
+            schema=schema,
+            drop=drop,
         )
-        vss.add_texts(texts=texts, metadatas=metadatas)
+        vss.add_texts(texts=texts, metadatas=metadatas, **kwargs)
         return vss
 
+    def delete(self, ids: Optional[List[str]] = None, **kwargs: Any) -> Optional[bool]:
+        """Delete vectors by uuids.
+
+        Args:
+            ids: List of ids to delete, where each id is a uuid string.
+        """
+        from psycopg2 import sql
+
+        if not ids:
+            return None
```

Comment:
out of curiosity why None instead of False

Reply:
I've taken the opportunity to expand the behavior to allow delete_all as well as delete by id. Returns None now. Actually, to your comment, return None is a nod to my DB background: delete successful, failed, or didn't happen :)